crates/plonk-verifier-foundry/.gitignore

```diff
@@ -0,0 +1,3 @@
+cache/
```

Comment:
This doesn't really belong to `crates`, right? Could you please create `contracts/` folder and move it there?